jkube-kit/jkube-kit-spring-boot/src/test/java/org/eclipse/jkube/springboot/enricher/AbstractSpringBootHealthCheckEnricherTestSupport.java

```diff
@@ -634,6 +634,73 @@ void testCustomPropertiesForLivenessAndReadiness() {
         assertHealthCheckDelays(livenessProbe, 460, 50, null);
     }
 
+    @Test
+    void testBuildProbeExplicitPathCreation_whenManagementHealthProbesEnabledIsTrue(){
+        props.put("management.health.probes.enabled","true");
+        writeProps();
+
+        Probe readinessProbe = new SpringBootHealthCheckEnricher(context).buildProbe(10, null, null, 3, 1,"/example");
+        assertHTTPGetPathAndPort(readinessProbe,getActuatorDefaultBasePath() + "/health/example", 8080);
+    }
+
+    @Test
+    void testBuildProbeDefaultPathCreation_whenManagementHealthProbesEnabledIsFalse(){
+        props.put("management.health.probes.enabled","false");
+        writeProps();
+
+        Probe readinessProbe = new SpringBootHealthCheckEnricher(context).buildProbe(10, null, null, 3, 1,"/example");
+        assertHTTPGetPathAndPort(readinessProbe,getActuatorDefaultBasePath() + "/health", 8080);
```

Comment:
I think there are tests already to cover the scenario where `management.health.probes.enabled` is `false` (I think `zeroConfig()` has same expectations as this test)

Reply:
I thought maybe it would be better to have separate test for the case where the management probe is explicitly kept as `false`.  If you think there's no need for such a testcase, then I can surely remove it.

Reply:
Umm, you do have a point. It might make sense to keep these tests in case we refactor `zeroConfig()` test. Having more tests doesn't do any harm.